README.md

```diff
@@ -61,7 +61,8 @@ ktlint {
 
 This plugin adds two tasks to every source set: `ktlint[source set name]Check` and `ktlint[source set name]Format`.
 Additionally, a simple `ktlintCheck` task has also been added that checks all of the source sets for that project.
-Similarly, a `ktlintFormat` task has been added that formats all of the source sets.
+Similarly, a `ktlintFormat` task has been added that formats all of the source sets. Also same meta tasks are added
+to the root project for convenience - they will call related tasks in subprojects. 
```

Comment:
Thanks! This is the first plugin I've published so it's very likely I did something goofy.